### ISSUE TITLE
registerar.schedule_code_upgrade doesn't work

### ISSUE THREAD

**Issue by xlc** [paritytech/polkadot#7202]
`registerar.schedule_code_upgrade` right now will update the para wasm and set upgrade go ahead signal. This doesn't work with Cumulus. Because Cumulus [requires `PendingValidationCode` is set](https://github.com/paritytech/cumulus/blob/e05c8d7f71734ed71188337c6cb0d30715f6320f/pallets/parachain-system/src/lib.rs#L396-L399) when go ahead signal is received. But because the upgrade it triggered directly on relaychain, `PendingValidationCode` is empty and therefore panic the chain.

`registerar.schedule_code_upgrade` shouldn't set go ahead signal and parachain can just perform client upgrade to force the new wasm.

**Comment by bkchr** [comment]
Yeah, `schedule_code_upgrade` doesn't make any sense. Basically we should merge `schedule_code_upgrade` and `force_code_upgrade`. It really doesn't make any sense to set the `GoAheadSignal`.
